action.yml

```diff
@@ -10,5 +10,5 @@ runs:
   steps:
     - shell: bash
       run: |
-        git config user.name "GitHub Actions"
-        git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+        git config --global user.name github-actions[bot]
+        git config --global user.email 41898282+github-actions[bot]@users.noreply.github.com
```

Comment:
Thank you! I'm not sure about the global change, this should be a separate PR if accepted at all. It might need to be opt-in and/or tested to make sure it works. I know some directories are read-only on GitHub Actions so this might just fault.